File: messages/src/main/java/com/thundashop/core/messagemanager/SmsMessage.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.thundashop.core.messagemanager;

import com.thundashop.core.common.DataCommon;

/**
 *
 * @author ktonder
 */
public class SmsMessage extends DataCommon {
    public String to = "";
    public String from = "";
    public String message = "";
    public String prefix = "";
    public String status = "not_delivered";
    public String response = "";
}

File: html5apps/terminal/events/core/messagemanager/SmsMessage.php
<?php
class core_messagemanager_SmsMessage extends core_common_DataCommon  {
	/** @var String */
	public $to;

	/** @var String */
	public $from;

	/** @var String */
	public $message;

	/** @var String */
	public $prefix;

	/** @var String */
	public $status;

	/** @var String */
	public $response;

	/** @var String */
	public $smsHander;

	/** @var String */
	public $outGoing;

}
?>